services/metadactyl-clj/src/metadactyl/user.clj

```diff
@@ -17,9 +15,9 @@
   (log/debug user-attributes)
   (let [uid (user-attributes :user)]
     (if (empty? uid)
-      (throw+ {:error_code common-errors/ERR_NOT_AUTHORIZED,
-               :user (dissoc user-attributes :password),
-               :message "Invalid user credentials provided."}))
+      (throw+ {:type :clojure-commons.exception/bad-request-field
+               :error "Invalid user credentials provided."
+               :user (dissoc user-attributes :password)}))
```

Comment:
(not directly related to your patch) The paranoid part of me would prefer this used `(select-keys user-attributes [... stuff we explicitly want to appear ...])` because there's always the possibility of adding more fields to something like this that should be private and missing the update.

(last job had an issue where password hashes got exposed because of something like this)